x-pack/plugins/cases/public/components/case_view/case_view_page.test.tsx

```diff
@@ -254,72 +219,30 @@ describe('CaseViewPage', () => {
       },
     }));
 
-    const result = appMockRenderer.render(<CaseViewPage {...caseProps} />);
+    appMockRenderer.render(<CaseViewPage {...caseProps} />);
 
-    expect(result.getByTestId('push-to-external-service')).toBeInTheDocument();
+    expect(await screen.findByTestId('edit-connectors')).toBeInTheDocument();
+    expect(await screen.findByTestId('push-to-external-service')).toBeInTheDocument();
 
-    userEvent.click(result.getByTestId('push-to-external-service'));
+    userEvent.click(screen.getByTestId('push-to-external-service'));
 
     await waitFor(() => {
       expect(pushCaseToExternalService).toHaveBeenCalled();
     });
   });
 
   it('should disable the push button when connector is invalid', async () => {
-    const result = appMockRenderer.render(
+    appMockRenderer.render(
       <CaseViewPage
         {...{
           ...caseProps,
           caseData: { ...caseProps.caseData, connectorId: 'not-exist' },
         }}
       />
     );
-    await waitFor(() => {
-      expect(result.getByTestId('push-to-external-service')).toBeDisabled();
-    });
-  });
 
-  it('should update connector', async () => {
```

Comment:
It was flaky. We already test this scenario in the `x-pack/plugins/cases/public/components/edit_connector/connectors_form.test.tsx` file.